Updated field lines

Piaero opened the pull request:
Adjusted terminology,
added 'rough' outside lines
added center line

Piaero committed:
Updated field lines

Adjusted terminology,
added 'rough' outside lines
added center line

dnikiel commented on main.lua:
```diff
@@ -198,9 +198,17 @@ function renderScore()
+function renderFieldOutline()
+  -- Field Outline
+  love.graphics.setLineStyle('rough')
+  love.graphics.setLineWidth( 2 )
```
No need for spacing here.

dnikiel commented on main.lua:
```diff
@@ -198,9 +198,17 @@ function renderScore()
+  -- Field Outline
+  love.graphics.setLineStyle('rough')
+  love.graphics.setLineWidth( 2 )
+  love.graphics.line(1,BOARD_BORDER_TOP,   VIRTUAL_WIDTH -1, BOARD_BORDER_TOP,    VIRTUAL_WIDTH -1 , VIRTUAL_HEIGHT -1,    1, VIRTUAL_HEIGHT-1,   1, BOARD_BORDER_TOP)
```
Please use single space after comma.

dnikiel commented on main.lua:
```diff
@@ -198,9 +198,17 @@ function renderScore()
+  -- Center line of the field
+  love.graphics.setLineStyle('smooth')
+  love.graphics.setLineWidth( 1 )
+  love.graphics.line(0.5 * VIRTUAL_WIDTH, BOARD_BORDER_TOP,   0.5 * VIRTUAL_WIDTH, VIRTUAL_HEIGHT)
```
Same comments as above.

Piaero committed:
Updated field lines

Adjusted terminology,
added 'rough' outside lines
added center line
formating comments took into account
